### PR TITLE
Tests: pam_pwhistory enforces root to password change history when re…

### PR DESCRIPTION
**Pull request by aborah-sudo**
…setting non-root user's password

https://bugzilla.redhat.com/show_bug.cgi?id=824858

### CODE REVIEW

File: multihost_test/bz_automation/script/bz824858.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+expect {
+    -nocase "passwd: all authentication tokens updated successfully." { exit 0}
+    -nocase "passwd: Have exhausted maximum number of retries for service" { exit 3}
+}
+expect eof
+exit 2
+EOF
+}
+
+do_passwd $1 $2
```
Comment (ikerexxe) [inline multihost_test/bz_automation/script/bz824858.sh:26]: Missing EOF

File: multihost_test/bz_automation/test_pam_pwhistory.py [hunk 2]
```diff
@@ -46,3 +47,41 @@ def test_2068461(self, multihost, create_localusers, bkp_pam_config):
...
+        execute_cmd(multihost, "touch /etc/security/opasswd")
+        execute_cmd(multihost, "chown root:root /etc/security/opasswd")
+        execute_cmd(multihost, "chmod 600 /etc/security/opasswd")
+        execute_cmd(multihost, "echo R3dh4T1nC | passwd --stdin pamtest1")
+        execute_cmd(multihost, "> /etc/security/opasswd")
+        execute_cmd(multihost, "sed -i -e 's/^password\s\+sufficient\s\+pam_unix.so/password"
+                               "    requisite     pam_pwhistory.so remember=3 "
+                               "use_authtok enforce_for_root\\n\\0/'  "
+                               "/etc/pam.d/system-auth")
+        file_localtion = "/multihost_test/bz_automation/script/bz824858.sh"
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_pam_pwhistory.py:72]: s/file_localtion/file_location

**Review by ikerexxe — COMMENTED** [review]
Just two minor comments and a big question: Would it be possible to implement the bash script in python? Or is it giving you some type of error like last time?

File: multihost_test/bz_automation/script/bz824858.sh (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/script/bz824858.sh]: Fixed

File: multihost_test/bz_automation/test_pam_pwhistory.py (hunk 2 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/test_pam_pwhistory.py]: Fixed

**Review by ikerexxe — APPROVED** [review]

**Pull request merged**